src/ssm.js

```diff
@@ -100,8 +103,14 @@
         fireAllMethodsInArray(leaveMethods);
         fireAllMethodsInArray(enterMethods);
         fireAllMethodsInArray(resizeMethods);
+
+        if (changed) {
```

Comment:
For the record, I was between doing this and `if (leaveMethods.length || enterMethods.length || resizeMethods.length)`, but figured this was more self-explanatory. Happy for it to go whichever way you prefer.